src/features/campaigns/components/ActivityList/FilterActivities.tsx

```diff
@@ -0,0 +1,90 @@
+import { ChangeEvent } from 'react';
+import { FilterListOutlined } from '@mui/icons-material';
+import {
+  Box,
+  Card,
+  Checkbox,
+  FormControlLabel,
+  FormGroup,
+  TextField,
+} from '@mui/material';
+
+import { ACTIVITIES } from 'features/campaigns/models/CampaignActivitiesModel';
+import messageIds from 'features/campaigns/l10n/messageIds';
+import { useMessages } from 'core/i18n';
+
+interface FilterActivitiesProps {
+  filters: ACTIVITIES[];
+  filterTypes: ACTIVITIES[];
+  onFiltersChange: (evt: ChangeEvent<HTMLInputElement>) => void;
+  onSearchStringChange: (evt: ChangeEvent<HTMLInputElement>) => void;
+  value: string;
+}
+
+const FilterActivities = ({
+  filters,
+  filterTypes,
+  onFiltersChange,
+  onSearchStringChange,
+  value,
+}: FilterActivitiesProps) => {
+  const messages = useMessages(messageIds);
+
+  return (
+    <Card>
+      <Box padding={2}>
+        <TextField
+          fullWidth
+          InputProps={{
+            startAdornment: (
+              <FilterListOutlined
+                color="secondary"
+                sx={{ marginRight: '0.5em' }}
+              />
+            ),
+          }}
+          onChange={onSearchStringChange}
+          placeholder={messages.singleProject.filterActivities()}
```

Comment:
I feel like the message structure does not reflect the fact that this is a reusable component that is used in at least two pages. If I understand this correctly, you found some existing strings (or group of strings) in the `messageIds.ts` module and used them, but they are probably unused or at least will be once we've finished implementing the new interface.

I think all strings in this component should be moved to it's own group (e.g. `feat.campaigns.activityFilter`).